pkg/generator/helper/helper.go

```diff
@@ -219,6 +219,14 @@ func run{{.StepName | title}}(config *{{ .StepName }}Options, telemetryData *tel
 
 // ProcessMetaFiles generates step coding based on step configuration provided in yaml files
 func ProcessMetaFiles(metadataFiles []string, stepHelperData StepHelperData, docuHelperData DocuHelperData) error {
+
+	path, _ := filepath.Split(metadataFiles[0])
+	fmt.Printf("Path: %v\n", path)
+	rootPath := strings.Replace(path, fmt.Sprintf("resources%vmetadata", string(filepath.Separator)), "", 1)
```

Comment:
Really just a suggestion, but somehow I needed to read twice before understanding:

```suggestion
	rootPath := strings.TrimPrefix(path, filepath.Join("resources", "metadata"))
```

Don't know if this is really equivalent... or if you need to also cut from the middle in some cases.